### DISCUSSION TITLE
[Feature] Bulk update datetime with a timedelta

### DISCUSSION THREAD

**Discussion by joeShuff** [immich-app/immich#9586] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

A few occasions I've had now where a bulk of photos needed to have their datetime changed by the same amount. (A camera import where the cameras time was incorrect). However the bulk update asset UI and endpoint only accepts an exact datetime and that will update every asset to that one time. 

I would like it if we could set a timedelta to apply to each asset. I.e "Move all these photos timestamp by +2 hours". 

I wrote a small script in python to fetch all items in an album then for each asset apply the delta and update that individual asset which worked just fine, but this would be easier in the web UI.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#9430
